modules/core/localizer.js

```diff
@@ -170,6 +179,20 @@ export function coreLocalizer() {
     function updateForCurrentLocale() {
         if (!_localeCode) return;
 
+        for (let localeCode of _localeCodes) {
+            if (localeCode in rematchCodes) {
+                let languageCodeList = rematchCodes[localeCode];
```

Comment:
this should be a `const` variable

```suggestion
                const languageCodeList = rematchCodes[localeCode];
```

similarly, also the two `for (const … of …)` loops